modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/Attachments.tsx

```diff
@@ -76,6 +76,21 @@ export function Attachments({setValues, values}: IProps) {
 		setAttachmentsFields(parseFields(fields));
 	};
 
+	useEffect(() => {
+		API.getAllObjectDefinitions().then((items) => {
+			const objectDefinitions = items.filter(({system}) => !system);
+
+			setObjectDefinitions(objectDefinitions);
+
+			const currentObjectDefinition = objectDefinitions?.find(
+				(item) => item.id === values.objectDefinitionId
+			);
+
+			setSelectedEntity(currentObjectDefinition);
+		});
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
```

Comment:
Hey @Artartur, you can also remove this `//eslint-disable-next-line react-hooks/exhaustive-deps`, hooks dependencies are very important for React, we use this comment in some cases that suggested dependencies are impacting what we want do, but it's not the case, you can put `values.objectDefinitionId` in the dependencies array